run.sh

```diff
@@ -316,8 +318,10 @@ if [[ ! -f "$PROJECT_ROOT/data/priv_validator_state.json" ]]; then
   echo '{"height":"0","round":0,"step":0}' > "$PROJECT_ROOT/data/priv_validator_state.json"
 fi
 
-if [ -n "$SNAPSHOT_PATH" ]; then
-  exec snapshot.sh "$START_CMD"
+if [ "$#" -ne 0 ]; then
+  exec $PREFIX_CMD "$@"
+elif [ -n "$START_CMD" ]; then
+  exec $PREFIX_CMD $START_CMD
 else
-  exec "$@"
+  exec $PREFIX_CMD start
```

Comment:
This isn't quite right - I assume it works for Cosmovisor, but otherwise it needs to be `$PROJECT_BIN start`. Currently errors if running without any arguments: `/usr/bin/run.sh: line 326: exec: start: not found`.

Reply:
Indeed, fixed (and pushed).